scripts/ag

```diff
@@ -277,7 +277,8 @@ def patch():
 def _create_database(force=False):
     # Connect to the postgres server
     try:
-        conn = connect(user=AMGUT_CONFIG.user, password=AMGUT_CONFIG.password,
+        conn = connect(dbname='postgres',
```

Comment:
I don't think that works - below is the error message when I make your suggested change

```
psycopg2.OperationalError: Cannot connect to the server, error is FATAL:  database "ag_testing2" does not exist
```

This is what I think is happening

psql client needs to connect to some database before it makes any changes, so it needs to connect to an existing database.  The above error is thrown because the `AMGUT_CONFIG.database` hasn't been created yet, so the psql client cannot connect to it. Since the `postgres` database is created by default, this is a good database to connect to before creating a new database.